src/java.management/share/classes/java/lang/management/ThreadMXBean.java

```diff
@@ -474,8 +475,8 @@ public interface ThreadMXBean extends PlatformManagedObject {
      * @param id the thread ID of a thread
      * @return the total CPU time for a thread of the specified ID
      * if the thread of the specified ID exists, the thread is alive,
-     * and CPU time measurement is enabled;
-     * {@code -1} otherwise.
+     * and CPU time measurement is enabled; {@code -1} if not enabled
+     * or the specified ID is a virtual thread
```

Comment:
It should be "{@code -1} if not enabled or the specified ID is a virtual thread or the thread does not exist or not alive."

Would this be simpler:

```
     * @return the total CPU time for a thread of the specified ID
     * if the thread of the specified ID is a platform thread, the thread is alive,
     * and CPU time measurement is enabled; {@code -1} otherwise.
```

I'm fine with either way.  Same comment for `getThreadUserTime(long)`

Reply:
That is a bit better as it avoids needing to list conditions for the "otherwise" case. I've update these methods to use that style and also updated the CSR.